### PR TITLE
feat: add support for clickhouse cloud

### PR DESCRIPTION
**Pull request by Ian2012**
## Description

This PR adds support for clickhouse cloud

## Useful links:

Depends on: 
- https://github.com/openedx/tutor-contrib-clickhouse/pull/16
- https://github.com/openedx/tutor-contrib-ralph/pull/20
- https://github.com/openedx/tutor-contrib-superset/pull/32

### CODE REVIEW

**Comment by Ian2012** [comment]
@bmtcril This PR must not be merged before #35 

We want this to be compatible with clickhouse cloud. Could you test it out?

https://github.com/openedx/tutor-contrib-oars/pull/35/files#diff-f362d0701f52ac1d6b980fa8bbb637c151d51743e350c60aa3f0e7aa50f4ba25R2-R6

Do you want me to open a proper issue here? https://github.com/openedx/openedx-event-sink-clickhouse

**Comment by Ian2012** [comment]
Here is an screenshot of the clickhouse cloud dashboard and superset events dashboard:

image

image

File: tutoroars/templates/oars/apps/pythonpath/superset-dashboard.py [hunk 1]
```diff
@@ -9,8 +9,10 @@
 from supersetapiclient.client import SupersetClient
 
 
-SUPERSET_URL_SCHEME = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}"
-SUPERSET_HOST_URL = f"{SUPERSET_URL_SCHEME}://{{SUPERSET_HOST}}{% if not ENABLE_HTTPS %}:{{ SUPERSET_PORT }}{% endif %}"
+SUPERSET_URL_SCHEME = (
+    "{% if ENABLE_HTTPS and not RUN_SUPERSET %}https{% else %}http{% endif %}"
```
Comment (mariajgrimaldi) [inline tutoroars/templates/oars/apps/pythonpath/superset-dashboard.py:13]: Why `not RUN_SUPERSET `?

File: tutoroars/templates/oars/apps/pythonpath/superset-dashboard.py [hunk 2]
```diff
@@ -9,8 +9,10 @@
...
+)
+SUPERSET_HOST_URL = SUPERSET_URL_SCHEME + "://superset:8088/"
```
Comment (mariajgrimaldi) [inline tutoroars/templates/oars/apps/pythonpath/superset-dashboard.py:15]: why //superset:8088/ hardcoded?

File: tutoroars/templates/oars/apps/pythonpath/superset-dashboard.py (hunk 1 above)
Comment (Ian2012) [reply tutoroars/templates/oars/apps/pythonpath/superset-dashboard.py:13]: To reduce network traffic, and avoid the TLS resolution

**Comment by Ian2012** [comment]
@bmtcril After further inspection, the other PR does not block this work.

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-oars&entry.867055334=https://github.com/openedx/tutor-contrib-oars/pull/38&entry.1484655318=https://github.com/Ian2012&entry.752974735=2023-05-08+18:07&entry.1917517419=2023-05-10+20:15&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**